examples/vault_integration/config/vault.yml

```diff
@@ -39,9 +48,14 @@ default_policies:
   - "$PROJECT-$TARGET"
   - "$PROJECT-$TARGET-$POD-$SERVICE"
 
-# If you want to apply addition policies to particular pods, you may also
+# If you want to apply additional policies to particular pods, you may also
```

Comment:
so do i need to mark every placeholder pod, or every placeholder service inside of every pod?

Reply:
Every placeholder service. The config file format has no way to apply attributes to entire pods without breaking old configs.

Besides, individual placeholders might very well want secrets at some point.

Reply:
ok that's fine thanks!